src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -7,7 +7,8 @@
 		<UseWPF>true</UseWPF>
 		<EnableDefaultPageItems>false</EnableDefaultPageItems>
 		<LangVersion>preview</LangVersion>
-		<AssemblyName>ComposeUI-Shell</AssemblyName>
+		<AssemblyName>MorganStanley.ComposeUI.Shell</AssemblyName>
+		<RootNamespace>MorganStanley.ComposeUI.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
```

Comment:
How about `<RootNamespace>MorganStanley.ComposeUI.Shell</RootNamespace>`? We might rename the project file, but we probably don't want the namespace to change as well.